### PR TITLE
Added keep-alive flag and functionality

### PR DESCRIPTION
**Pull request by prayuj**
This relates to [Issue #17](https://github.com/abhinavdc/cowin-pinger/issues/17)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -146,15 +153,25 @@ function pingCowin({ key, hook, age, districtId, appointmentsListLimit, date, pi
...
+                    }
                 });
             } else {
                 console.log(dataOfSlot);
-                console.log('Slots found\nStopping Pinger...')
-                sound.play(notificationSound, 1);
-                clearInterval(timer);
+                if (keepAlive) {
+                    console.log('Slots found')
+                    sound.play(keepAliveNotificationSound, .8);
```
Comment (abhinavdc) [inline app.js:169]: This notification sound is a little too short and would be easy to miss. Let's keep the existing one itself or  a longer notification sound if you can find one.

**Comment by abhinavdc** [comment]
All except the notification sound looks good. Let's merge it in after changing it.

Also thanks for the PR @prayuj

File: app.js (hunk 1 above)
Comment (prayuj) [reply app.js:169]: I added a new tone to differentiate keep alive flow and normal flow. I'll revert it back to the original, that makes more sense

**Comment by prayuj** [comment]
@abhinavdc All done!

**Comment by prayuj** [comment]
Now I am done!

**Comment by abhinavdc** [comment]
Awesome work @prayuj. Thank you! Let's merge it.

**Pull request merged**